tests/cql/CqlListOperatorsTest.xml

```diff
@@ -26,6 +26,26 @@
 			<expression>({ DateTime(2012, 10, 5, 10), DateTime(2012, 1, 1), DateTime(2012, 1, 1, 12), DateTime(2012, 10, 5) }) S sort desc</expression>
 			<output>{DateTime(2012, 10, 5, 10), DateTime(2012, 10, 5), DateTime(2012, 1, 1, 12), DateTime(2012, 1, 1)}</output>
 		</test>
+		<test name="intList">
+			<expression>{ 3, 2, 1 }</expression>
+			<output>{3, 2, 1}</output>
+		</test>
+		<test name="decimalList">
+			<expression>{ 3.8, 2.4, 1.9 }</expression>
+			<output>{3.8, 2.4, 1.9}</output>
+		</test>
+		<test name="quantityList">
+			<expression>{ 19.99 'lbs', 17.33 'lbs', 10.66 'lbs' }</expression>
+			<output>{19.99 'lbs', 17.33 'lbs', 10.66 'lbs'}</output>
+		</test>
+		<test name="dateTimeList">
+			<expression>{ DateTime(2016), DateTime(2015), DateTime(2010) }</expression>
+			<output>{@2016, @2015, @2010}</output>
+		</test>
+		<test name="timeList">
+			<expression>{ @T15:59:59.999, @T15:12:59.999, @T15:12:13.999 }</expression>
+			<output>{@T15:59:59.999, @T15:12:59.999, @T15:12:13.999}</output>
+		</test>
```

Comment:
All of this was mistakenly added to a duplicated "Sort" group at the end of the file.  I've added the tests to the original "Sort" group here and removed the duplicate at the end of the file.